### PR TITLE
Lmaoo169 1

### PR DESCRIPTION
**Pull request by LewisWelch94**

### CODE REVIEW

File: Script/projectController.js [hunk 1]
```diff
@@ -79,18 +79,26 @@ function getTicketWithProjectId(id)
 
 function createProjectPrompt()
 {
+    document.getElementById("projectModalHead").innerHTML = `
+            Project Creation Modal
```
Comment (OnikNoor97) [inline Script/projectController.js:83]: Use Speech Marks, also dont call it Modal, call it "Create Project"

File: Script/projectController.js [hunk 2]
```diff
@@ -138,17 +146,23 @@ function createProject()
 
 function createTicketPrompt(projectId)
 {
+    document.getElementById("projectModalHead").innerHTML = `
+            Ticket Creation Modal
```
Comment (OnikNoor97) [inline Script/projectController.js:150]: Create Ticket, also use speech marks, use single line to make code tidy

**Comment by OnikNoor97** [comment]
Leaving CSS changes to Tufan, seems good at the moment

File: Script/projectController.js (hunk 1 above)
Comment (LewisWelch94) [reply Script/projectController.js]: Done

File: Script/projectController.js [hunk 3]
```diff
@@ -79,18 +79,24 @@ function getTicketWithProjectId(id)
 
 function createProjectPrompt()
 {
+    document.getElementById("projectModalHead").innerHTML = ` "Create Project" `;
```
Comment (OnikNoor97) [inline Script/projectController.js:82]: Remove String Literals

File: Script/projectController.js [hunk 4]
```diff
@@ -138,17 +144,21 @@ function createProject()
 
 function createTicketPrompt(projectId)
 {
+    document.getElementById("projectModalHead").innerHTML = ` "Create Ticket" `;
```
Comment (OnikNoor97) [inline Script/projectController.js:147]: Same as above

File: Css/allModals.css [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+	background: #396060;
+}
+
+.modal-title-custom {
+	margin-left: 125px;
+	margin-right: auto;
+	color: white;
+}
+
+.projectModalSelection1 {
```
Comment (OnikNoor97) [inline Css/allModals.css:20]: Modal Selection 1? Not sure what you mean here, name it specifically to what modal you are referring to

File: Css/allModals.css [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+	margin-top: 10px;
+	margin-bottom: 10px;
+}
+
+.projectModalSelection2 {
```
Comment (OnikNoor97) [inline Css/allModals.css:25]: Same as abovd

File: Global/editUserModal.php [hunk 7]
```diff
@@ -1,12 +1,12 @@
 <?php if (!isset($userLoggedIn))  { return; } else { ?>
-<link rel="stylesheet" href="../Css/editUserModal.css">
+<link rel="stylesheet" href="../Css/allModals.css">
```
Comment (OnikNoor97) [inline Global/editUserModal.php:2]: Good approach. I like this. Saves CSS code

**Pull request merged**